app/components/NetworkStatus/NetworkStatus.tsx

```diff
@@ -39,6 +43,19 @@ const getStartupStatusText = (startupStatus: NodeStartupState) => {
       return 'Starting GRPC server...';
     case NodeStartupState.VerifyingLayers:
       return 'Tortoise verifying layers...';
+    case NodeStartupState.SyncingAtxs:
+      return (
+        <>
+          Syncing Activations...
+          <Tooltip
+            width={200}
+            marginTop={-2}
+            text="This process may take hours and consume CPU. Please, wait for it."
```

Comment:
I think that requires better wording. Ideas @lrettig ?

Reply:
How about: 
`Please wait. This process may require several hours and increased CPU usage to complete.`